.gitignore

```diff
@@ -1,3 +1,4 @@
+Gemfile.lock
```

Comment:
I talked with some other Ruby coders about this and here is another reference about it:
http://stackoverflow.com/questions/4151495/should-gemfile-lock-be-included-in-gitignore